modules/core/src/test/java/org/apache/ignite/internal/processors/cache/GridCacheAbstractMetricsSelfTest.java

```diff
@@ -209,8 +206,7 @@ public void testGetMetricsDisable() throws Exception {
         // Invoke update on cache.
         for (int i = 0; i < KEY_CNT; i++)
             jcache.invoke(i, new CacheEntryProcessor<Object, Object, Object>() {
-                @Override
-                public Object process(MutableEntry<Object, Object> entry,
+                @Override public Object process(MutableEntry<Object, Object> entry,
```

Comment:
Here and below -- each argument must be on a new line.

Reply:
fixed